src/layout/components/SocialIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+type SocialIcons = {
+   href: string;
+   icon?: any;
+   text?: string;
+   extraClasses?: string;
+};
+
+export const SocialIcon: React.FC<SocialIcons> = ({ href, icon, text, extraClasses }) => (
+   <li className="footer-link-item hover:-translate-y-2 text-white hover:text-white duration-200">
+      <a href={href} className={`${extraClasses}`}>
+         <FontAwesomeIcon icon={icon} className=" w-10 h-10" />
+         {text}
+      </a>
+   </li>
+);
```

Comment:
You can name this property `clasName` and use it like this:

```suggestion
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

type SocialIcons = {
   href: string;
   icon?: any;
   text?: string;
   className?: string;
};

export const SocialIcon = ({ href, icon, text, className }: SocialIcons) => (
   <li className="footer-link-item hover:-translate-y-2 text-white hover:text-white duration-200">
      <a href={href} className={className}>
         <FontAwesomeIcon icon={icon} className=" w-10 h-10" />
         {text}
      </a>
   </li>
);
```